doc/source/whatsnew/v0.25.0.rst

```diff
@@ -68,6 +68,8 @@ Performance Improvements
 Bug Fixes
 ~~~~~~~~~
 
+- Bug in using duplicated with empty dataframes (:issue:`25184`)
```

Comment:
say more like

``:meth:DataFrame.duplicate()`` on empty dataframe was not return a boolean dtyped Series

Reply:
this can go in 0.24.2